src/registrar/assets/sass/_theme/styles.scss

```diff
@@ -9,3 +9,7 @@
 /*--------------------------------------------------
 --- Custom Styles ---------------------------------*/
 @forward "uswds-theme-custom-styles";
+
+/*--------------------------------------------------
+--- Admin ---------------------------------*/
+@forward "admin";
```

Comment:
endline is off here. running black . should format it, but I also thought Brandon added something to auto add the newlines when needed.

Reply:
Neither did anything. Maybe scss is ignored? Anyways, I'm adding an endline manually.